build/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+HOST_DIR=$(cd $(dirname "$0")/../; pwd)
+CONTAINER_DIR="/opt/rook.github.io"
+
+# install node dependencies through container
+echo "pulling npm dependencies..."
+docker run -t --user 1000 -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine npm install
+
+# run prepare script through container
+echo "preprocessing static assets..."
+docker run -t --user 1000 -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine node build/scripts/preprocess.js
```

Comment:
I think it must be a fluke.  Perhaps the mapping only occurs if the given id is legit and maps to someone.  Also, running the command without user mapping at all worked on mac, but did not on ubuntu originally.

Open to suggestions on how to handle this locally on mac with the correct `--user $(id -u):$(id -g)` -- perhaps only pass this flag if non-mac ?